openqabot/openqabot.py

```diff
@@ -3,14 +3,15 @@
 from argparse import Namespace
 from logging import getLogger
 from os import environ
+import requests as req
```

Comment:
why ?

Reply:
To distinguish from the internal requests I added